LLIO2025/src/services/ActivityApiService.ts

```
@@ -3,94 +1,123 @@
+  }
+};
+
+const getAllActivitesFromRange = async (startDate: string, endDate: string) => {
+  try {
+    const response = await GET<{ activities: RawActivity[] }>(
+      '/activities/me/' + startDate + '/' + endDate
+    );
+    console.log('response', response);
+
+    // Vérifier si la réponse contient la propriété 'activities' et si c'est un tableau
+    if (response && response.activities && Array.isArray(response.activities)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)